lld/MachO/BPSectionOrderer.cpp

```diff
@@ -60,12 +60,13 @@ getRelocHash(const Reloc &reloc,
   return getRelocHash(kind, sectionIdx.value_or(0), 0, reloc.addend);
 }
 
-static void constructNodesForCompression(
-    const SmallVector<const InputSection *> &sections,
+static SmallVector<
```

Comment:
Can you add comments to this function about:
- What is the return value
- What is `Uns`